resources/js/chat/chat-state-store.ts

```diff
@@ -47,24 +50,47 @@ export default class ChatStateStore implements DispatchListener {
 
   // Should not join/create another channel if still waiting for a pending request.
   @computed
-  get isJoiningChannel() {
-    return this.waitJoinChannelUuid != null;
+  get isAddingChannel() {
```

Comment:
is it actually useful having `@computed` for simple comparison like this and some others here